onnxruntime/contrib_ops/cuda/math/bias_softmax_impl.cu

```diff
@@ -0,0 +1,314 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/common.h"
+#include "core/providers/cuda/cuda_common.h"
+#include "core/providers/cuda/cudnn_common.h"
+#include "core/providers/cuda/cu_inc/common.cuh"
+#include "core/providers/cuda/math/softmax_impl.cuh"
+#include "core/providers/cuda/cu_inc/binary_elementwise_impl.cuh"
+#include "core/providers/cuda/math/binary_elementwise_ops_impl_functors.cuh"
+#include "bias_softmax.h"
```

Comment:
full path, put this one first per https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes